sagemaker-pipelines/tabular/sklearn-scheduled-training-and-inference/scheduled-sm-pipelines-sklearn.ipynb

```diff
@@ -0,0 +1,697 @@
+{
```

Comment:
<p>SageMaker Python SDK is installed in SageMaker notebooks and Studio by default.&nbsp;</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2755/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>